libraries/WiFi/examples/WiFiClient/README.md

```diff
@@ -0,0 +1,164 @@
+Access-Control-Allow-Headers: origin, content-type, X-Requested-With
+Access-Control-Allow-Methods: GET, POST, PUT, OPTIONS, DELETE, PATCH
+ETag: W/"5feceb66ffc86f38d952786c6d696c79"
+X-Frame-Options: SAMEORIGIN
+
+0
+Closing connection
+```
+Last portion is the read log:
+```
+HTTP/1.1 200 OK
+Date: Fri, 13 Jan 2023 13:12:32 GMT
+Content-Type: application/json; charset=utf-8
+Transfer-Encoding: chunked
+Connection: close
+Status: 200 OK
+Cache-Control: max-age=7, private
+Access-Control-Allow-Origin: *
+Access-Control-Max-Age: 1800
+X-Request-Id: 91b97016-7625-44f6-9797-1b2973aa57b7
+Access-Control-Allow-Headers: origin, content-type, X-Requested-With
+Access-Control-Allow-Methods: GET, POST, PUT, OPTIONS, DELETE, PATCH
+ETag: W/"8e9c308fe2c50309f991586be1aff28d"
+X-Frame-Options: SAMEORIGIN
+
+1e3
+{"channel":{"id":2005329,"name":"WiFiCLient example","description":"Default setup for Arduino ESP32 WiFiClient example","latitude":"0.0","longitude":"0.0","field1":"data0","created_at":"2023-01-11T15:56:08Z","updated_at":"2023-01-13T08:13:58Z","last_entry_id":2871},"feeds":[{"created_at":"2023-01-13T13:11:30Z","entry_id":2869,"field1":"359"},{"created_at":"2023-01-13T13:11:57Z","entry_id":2870,"field1":"361"},{"created_at":"2023-01-13T13:12:23Z","entry_id":2871,"field1":"363"}]}
+0
+
+
+Closing connection
+```
+After this the write+read log repeat every 10 seconds.
+
+
+## Troubleshooting
+
+***Important: Make sure you are using a good quality USB cable and that you have a reliable power source***
+
+* **WiFi not connected:** Check the SSID and password and also that the signal has sufficient strength.
+* **400 Bad Request:** Check the writeApiKey.
+* **404 Not Found:** Check the channel ID.
+* **No data on chart / reading NULL:** Data must be sent as an integer, without commas.
+
+If the error persists, you can ask for help at the official [ESP32 forum](https://esp32.com) or see [Contribute](#contribute).
+
+## Contribute
+
+To know how to contribute to this project, see [How to contribute.](https://github.com/espressif/arduino-esp32/blob/master/CONTRIBUTING.rst)
+
+If you have any **feedback** or **issue** to report on this example/library, please open an issue or fix it by creating a new PR. Contributions are more than welcome!
+
+Before creating a new issue, be sure to try the Troubleshooting and to check if the same issue was already created by someone else.
+
+## Resources
+
+* Official ESP32 Forum: [Link](https://esp32.com)
+* Arduino-ESP32 Official Repository: [espressif/arduino-esp32](https://github.com/espressif/arduino-esp32)
+* ESP32 Datasheet: [Link to datasheet](https://www.espressif.com/sites/default/files/documentation/esp32_datasheet_en.pdf)
+* ESP32-S2 Datasheet: [Link to datasheet](https://www.espressif.com/sites/default/files/documentation/esp32-s2_datasheet_en.pdf)
+* ESP32-C3 Datasheet: [Link to datasheet](https://www.espressif.com/sites/default/files/documentation/esp32-c3_datasheet_en.pdf)
```

Comment:
Please add ESP32-S3 datasheet link too, already asked to add it into the readme template as there is missing.